packages/fether-electron/src/main/app/options/index.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import extend from 'extend';
+
+import { DEFAULT_OPTIONS, TASKBAR_OPTIONS } from './config';
+
+let hasCalledInitFetherAppOptions = false;
+
+class FetherAppOptions {
+  options = {};
+
+  create = (withTaskbar, customOptions) => {
```

Comment:
This looks a lot like a constructor. Any reason why you're using a method instead?

Reply:
Sorry I was confused. Using a constructor appears to work. But it says they've deprecated constructors in Babel 7 in favour of decorators (i.e. babel-plugin-transform-class-constructor-call) https://babeljs.io/docs/en/babel-plugin-transform-class-constructor-call.
Here it shows usage with constructors too https://www.npmjs.com/package/@babel/plugin-transform-classes

Reply:
> But it says they've deprecated constructors in Babel 7 in favour of decorators (i.e. babel-plugin-transform-class-constructor-call) https://babeljs.io/docs/en/babel-plugin-transform-class-constructor-call.

No, they've deprecated constructors calls:
```
const d1 = new Date(); // allowed
const d2 = Date(); // forbidden
```